docs/configuration/reference/apis.md

```diff
@@ -0,0 +1,85 @@
+# `apis`
+
+## Introduction
+
+If your project contains multiple APIs, the `apis` configuration section allows you to set up different rules and settings for different APIs.
+
+## Options
+
+{% table %}
+
+- Option
+- Type
+- Description
+
+---
+
+- `{name}@{version}`
+- [API object](#api-object)
+- Required. Each API needs a name and optionally a version. Supports alphanumeric characters and underscores.
```

Comment:
Should we make the Required all caps and bolded as in the other pages in the beta docs?

Reply:
Updated this everywhere.